src/test/java/com/ioet/bpm/bpmorganizationsapi/BpmorganizationsapiApplicationTests.java

```diff
@@ -22,6 +24,9 @@
     @Mock
     OrganizationRepository repository;
 
+    @Mock
+    OrganizationController organizationController;
+
     @Test
     public void contextLoads() {
     }
```

Comment:
What's is doing this method here?

Reply:
This is generated by the IDE. I did forgot to delete it.